docs_sphinx/user/computation.rst

```diff
@@ -43,12 +43,11 @@ generation process is set in the `codegen.target` preference. By default, this
 preference is set to ``'auto'``, meaning that it will chose a compiled language
 target if possible and fall back to Python otherwise (it will also raise a warning
 in this case, set `codegen.target` to ``'numpy'`` explicitly to avoid this warning).
-There are two compiled language targets for Python 2.x, ``'weave'`` (needing a
-working installation of a C++ compiler) and ``'cython'`` (needing the `Cython`_
-package in addition); for Python 3.x, only ``'cython'`` is available. If you want to
+The compiled language target is ``'cython'`` (needing the `Cython`_ package in
+addition). If you want to
```

Comment:
This should say needing Cython and a working C++ compiler.

Reply:
Done!